### PR TITLE
IGNITE-15965 Fix a race between cluster start and setting the local member

### PR DESCRIPTION
**Pull request by sashapolo**
https://issues.apache.org/jira/browse/IGNITE-15965

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -68,7 +68,7 @@
         <spoon.framework.version>8.4.0-beta-18</spoon.framework.version>
         <typesafe.version>1.4.1</typesafe.version>
         <hamcrest.version>2.2</hamcrest.version>
-        <scalecube.version>2.6.6</scalecube.version>
+        <scalecube.version>2.6.12</scalecube.version>
```
Comment (SammyVimes) [inline parent/pom.xml:71]: While you're at it: could you please remove the `stopJmxMonitor` method from ScaleCubeClusterServiceFactory.java? :) This should be done by the scalecube itself in this version.

Comment (sashapolo) [reply parent/pom.xml:71]: I've already done that, that's why I updated the ScaleCube version in the first place

Comment (SammyVimes) [reply parent/pom.xml:71]: Oops, sorry)

**Review by SammyVimes — APPROVED** [review]
Looks good to me

**Pull request merged**